### PR TITLE
Increase default max-rps for tap and top

### LINKED ISSUE #1525: Tap sampling is windowed and that is odd
When using the tap defaults against emojivoto/web, I don't see a stream of requests (instead a bucket every 10s). It is a little odd to see 10 requests come in all at once via. tap even though they were evenly spread throughout the 10s window.

Assuming a `--max-rps 1`, I would expect to see a single request immediately with any others in that one second bucket suppressed.

### PR DESCRIPTION
**Pull request by adleong**
The default value for the max-rps argument to the tap and top commands is an overly conservative 1rps.  This causes the data to come in very slowly and much data to be discarded.  Furthermore, because tap requests are windowed to 10 seconds, this causes long pauses between updates.

We fix this in two ways.  Firstly we reduce the window size to 1s so that updates will come in at least once per second, even when the actual RPS of the data path is extremely high.  Secondly, we increase the default max-rps parameter from 1 to 100.  This allows tap to paint an accurate picture of the data much more quickly and sidesteps some sampling bias that happens when the max-rps is low.

In general, tap events tend to happen in bursts.  For example, one request in may trigger one or more requests out.  Likewise, a single upstream event may trigger several requests to the tapped pod in quick succession.  Sampling bias will occur when the max-rps is less than the actual rps and when the tap event limit subdivides these event bursts (biasing towards the first few events in the burst).  The greater the max-rps, the less the effects of this bias.

Fixes #1525 

Signed-off-by: Alex Leong <alex@buoyant.io>

### CODE REVIEW

File: controller/tap/server.go [hunk 1]
```diff
@@ -240,8 +240,8 @@ func destinationLabels(resource *public.Resource) map[string]string {
 // request is cancelled via the context.  Thus it should be called as a
 // go-routine.
 // To limit the rps to maxRps, this method calls Observe on the pod with a limit
-// of maxRps * 10s at most once per 10s window.  If this limit is reached in
-// less than 10s, we sleep until the end of the window before calling Observe
+// of maxRps * 1s at most once per 10s window.  If this limit is reached in
```
Comment (olix0r) [inline controller/tap/server.go]: "at most once per" _1s window?_

Comment (adleong) [reply controller/tap/server.go]: yup, thank you

**Comment by olix0r** [comment]
Do i understand correctly that this will cause the tap server to issue new requests to all proxies once per second? This seems potentially... costly, especially if the user intends to view results for more than 1s. I suspect we'll need to push down sampling deeper into the proxy

**Comment by adleong** [comment]
Yes, that's correct.  This will cause the tap server to issue a new tap request to the proxy/proxies it is tapping once per second.  I don't have a good sense for how costly this really is.

**Comment by adleong** [comment]
Actually, let me clarify.  The tap server will wait a *minimum* of 1 second between sending tap requests.  If the limit (now defaulting to 100) has not been reached after 1 second, the tap request will remain open and a new tap request will not be made until 100 events have been received.  

It's only in the case where the limit is reached in less than 1 second that the tap server will make a new request after the 1 second window.

**Comment by adleong** [comment]
Tested this at 200rps and it behaved as I expected: each second we get 100 requests and then pause until the next second.

**Pull request merged**